app/server.py

```diff
@@ -11,7 +11,13 @@
 ######################################################################
 @app.route('/')
 def index():
-    """ Root URL response """
+    """
+    User interface
+    ---
+    tags:
+      - Shopcarts
+    description: The Shopcarts endpoint allows you to query shopcarts
+    """
```

Comment:
I don't really understand why this is here - this route is just returning a static file for the UI. You can see the professor's corresponding `/` handler [here](https://github.com/nyu-devops/lab-flask-swagger/blob/2a16bfe0a595e035261eee306bcaf43d2aeb2bbd/server.py#L73)